src/components/ListItem.jsx

```diff
@@ -40,7 +41,6 @@ export function ListItem({ name, itemId, dateLastPurchased }) {
 
 	// When remove button is clicked, a confirm window pops up, and when user confirms, deleteItem() is called
 	const handleDelete = async (e) => {
-		const listId = localStorage.getItem('tcl-shopping-list-token');
 		if (window.confirm(`Do you want to remove ${name}?`)) {
```

Comment:
FWIW, you could have also written just `confirm(...)` here since the window object is the global namespace.